packages/metascraper-amazon/src/index.d.ts

```diff
@@ -0,0 +1 @@
+export default function rules(): import('metascraper').Rules;
```

Comment:
Because you wrote `module.exports = () => {}`, these should be of the form:

```ts
declare function rules(): ...;

export = rules;
```

Not export default. You'll only want to export default if you were using ESM.